### PR TITLE
Add logging commands

### PR DESCRIPTION
**Pull request by gabriel-samfira**
This change enables log download and streaming from coriolis-logger.

Please see:

```bash
coriolis log --help
```

for usage info

### CODE REVIEW

File: coriolisclient/cli/logging.py [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+
+    def get_parser(self, prog_name):
+        parser = super(DownloadCoriolisLog, self).get_parser(prog_name)
+        parser.add_argument(
+            'log_name',
+            help='The name of the log to fetch')
+        parser.add_argument(
+            '--start-time',
+            help="The start date of the log. This can be a unix timestamp"
+            " or a period written as one of the following:"
```
Comment (aznashwan) [inline coriolisclient/cli/logging.py]: NIT: align these string continuations with the `=`

File: coriolisclient/cli/shell.py [hunk 2]
```diff
@@ -29,8 +29,9 @@
 from cliff import help
 from keystoneauth1 import loading
 from keystoneauth1 import session
-from keystoneclient.auth.identity import v3
-from keystoneclient.auth.identity import v2
+from keystoneauth1.identity import v3
+from keystoneauth1.identity import v2
```
Comment (aznashwan) [inline coriolisclient/cli/shell.py:32]: NIT: import v3 after v2

File: coriolisclient/v1/logging.py [hunk 3]
```diff
@@ -0,0 +1,182 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import asyncio
+import datetime
+import json
+import requests
+import websockets
+from six.moves.urllib import parse as urlparse
+from keystoneauth1.exceptions.catalog import EndpointNotFound
```
Comment (aznashwan) [inline coriolisclient/v1/logging.py:23]: NITS:
* `websockets`, `six` and `keystoneauth` should be in a separate block for 3rd party packages
* `six` would come after `keystoneauth`

**Review by aznashwan — APPROVED** [review]
LGTM. Only a couple of minor nits which are safe to ignore.

**Comment by gabriel-samfira** [comment]
Done

**Comment by aznashwan** [comment]
LGTM. Merging. Thanks!

**Pull request merged**